bin/verify

```diff
@@ -144,6 +145,19 @@ verify_email_config() {
     chmod 600 $HOME/asgs-global.conf && \
       echo "ok, permissions fixed for $HOME/asgs-global.conf"
   fi
+  if [ -n "$TESTTO" ]; then
+    echo ++ Sending a test email to: $TESTTO
+    cat $(which asgs-sendmail) | asgs-sendmail --to $TESTTO --subject "ASGS email send verification test"
+    err=$?
+    if [ $? == 0 ]; then
+      echo "ok, no errors detected during send; check inbox for test email"
+    else
+      echo "not ok, sending email via asgs-sendmail resulted in an exit code of $err"
```

Comment:
This error message should probably be preceded with `warning:` to be consistent.

Reply:
This is mimicking the "TAP" output of ok/not ok and is meant for parsing; thank you for pointing that out. I can explain next time we talk.